### ISSUE TITLE
Undefined variable 's' in Precheck method

### ISSUE THREAD

**Issue by rutgervandriel** [Arg0s1080/mrz#26]
Hello, 

I ran into the following bug when calling the precheck method:
```
raise LengthError(cause=len(s), document=document_description, length=length)
NameError: name 's' is not defined
```

Having a look at precheck method in [string_checkers.py in 0.5.8](https://github.com/Arg0s1080/mrz/blob/v0.5.8/mrz/base/string_checkers.py#L98) I observe the following:
```

def precheck(document_description: str, string: str, length: int):
    s = string.replace("\n", "")
    if _is_string(_check_upper(string)) and len(s) != length:
        raise LengthError(cause=len(s), document=document_description, length=length)
    if not is_printable(s):
        raise FieldError("%s contains invalid characters" % document_description, s)
```
Having a look at precheck method in [string_checkers.py in 0.6.1](https://github.com/Arg0s1080/mrz/blob/v0.6.1/mrz/base/string_checkers.py#L98) I observe the following:
```
def precheck(document_description: str, string: str, length: int):
    if check_string(_check_upper(string)) and len(string) != length:
        raise LengthError(cause=len(s), document=document_description, length=length)
    if not is_printable(string, "\n"):
        raise FieldError("%s contains invalid characters" % document_description, s)
```
Since the removal of `s = string.replace("\n", "")` there is no defined variable _s_ anymore. Both the LengthError and FieldError reference to this. I believe it should be _string_ instead of _s_ now.

**Comment by Arg0s1080** [comment]
Hi!

Its true! Honestly i dont know what happened. Quite few things have been changed. Some error must have occurred during the process. My bad...

Fixed and released v0.6.2

Thank you very much

**Comment by rutgervandriel** [comment]
No worries, thanks for the fix!
